templates/layout.html

```diff
@@ -89,11 +89,21 @@ <h2 class="tagline">
     </div>
 
     {% if config.get('USE_MATOMO') %}
-      <noscript><img src="https://{{ config.get('MATOMO_URL') }}/js/?idsite={{ config.get('MATOMO_SITE_ID') }}&amp;rec=1&amp;action_name=Panopticlick{% if title %} | {{ title }}{% endif %}" style="border:0" height="0" width="0" alt="" /></noscript>
-      <div style="height: 0; width: 0; position: absolute" id="anon-stats"></div>
+      <!-- Matomo -->
       <script type="text/javascript">
-        document.getElementById('anon-stats').innerHTML = '<img src="https://{{ config.get('MATOMO_URL') }}/js/?idsite={{ config.get('MATOMO_SITE_ID') }}&amp;rec=1&amp;urlref=' + encodeURIComponent(document.referrer) + '&amp;action_name=' + encodeURIComponent(document.title) + '" style="border:0" height="0" width="0" alt="" />';
+        var _paq = window._paq = window._paq || [];
+        /* tracker methods like "setCustomDimension" should be called before "trackPageView" */
+        _paq.push(['trackPageView']);
+        _paq.push(['enableLinkTracking']);
+        (function() {
+          var u="https://anon-stats.eff.org/";
+        _paq.push(['setTrackerUrl', u+'piwik.php']);
+        _paq.push(['setSiteId', '40']);
```

Comment:
This needs to be generalized to `config.get('MATOMO_SITE_ID')`

Reply:
Okay, I usually just use the code provided by anon-stats, but it seems I should leave the code that was already there.  I'm not understanding where the MATOMO_SITE_ID AND MATAMO_URL are being set though. I see it referenced in main.py and env_config.py.

Reply:
It should be set in the deployment repo environmental variables, I'll message you with the details.